manifests/c/CoolPlayLin/Installer/LANDrop/0.4.0/CoolPlayLin.Installer.LANDrop.installer.yaml

```
@@ -0,0 +1,32 @@
+# Created with komac v2.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: CoolPlayLin.Installer.LANDrop
+PackageVersion: 0.4.0
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Platform:
+  - Windows.Desktop
+Dependencies:
+  PackageDependencies:
+    - PackageIdentifier: Microsoft.VCRedist.2015+.x86
+UpgradeBehavior: install
+ReleaseDate: 2024-01-23
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/CoolPlayLin/Windows-Setups/releases/download/v0.0.2/LANDrop.Setup.ia32.exe
+  InstallerSha256: 5022C23DF12627539497C94F629CACF34E6AC4DA2D3AA824B50B8B31FFA55951
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86

```

```suggestion
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
UpgradeBehavior: install
ReleaseDate: 2024-01-23
Installers:
- Architecture: x64
  Scope: user
  InstallerUrl: https://github.com/CoolPlayLin/Windows-Setups/releases/download/v0.0.2/LANDrop.Setup.ia32.exe
  InstallerSha256: 5022C23DF12627539497C94F629CACF34E6AC4DA2D3AA824B50B8B31FFA55951
  InstallerSwitches:
    Custom: /CURRENTUSER
- Architecture: x64
```

Hi @CoolPlayLin,

Was this package compiled for 32-bit OSes? From the error above, it sounds like this package is 64-bit. Our `x86` validations are performed on a 32-bit OS.